### PR TITLE
Add "DarkScience" to default server list.

### PR DESCRIPTION
**Pull request by dijit**
According to [netsplit.de's top 100 list](https://netsplit.de/networks/top100.php) "DarkScience" is ranked 66 for number of users.

Notably hosting the [Hak5](https://hak5.org) and [joinmarket](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/293/files#diff-4f3c2beacd536175f30616463ddaeb33) communities, along with the social aspect of the network itself.

The environment surrounding the network is one of good standing and has a fair reputation of legal and ethical behaviour.

I think this is a fair addition to the list, please let me know if there are any concerns. :)

### CODE REVIEW

File: src/common/servlist.c [hunk 1]
```diff
@@ -119,6 +119,11 @@ static const struct defaultserver def[] =
 
 	{"DarkMyst", 0, 0, 0, LOGIN_SASL, 0, TRUE},
 	{0,			"irc.darkmyst.org"},
+	
+	{"darkscience", 0, 0, 0, LOGIN_SASL, 0, TRUE},
```
Comment (TingPing) [inline src/common/servlist.c:123]: Wrap in `#ifdef USE_OPENSSL` like some of the other networks.

Fix whitespace to match others.

Comment (dijit) [reply src/common/servlist.c:124]: Thanks for the feedback, I am not sure how the whitespace got messed up (I was editing using github web).

I have made the changes you suggested, the entry will be completely removed if openssl is not available, as the network does not allow plaintext. I hope that is ok.

Comment (TingPing) [reply src/common/servlist.c:124]: You are mixing spaces when the file uses tabs.

Comment (dijit) [reply src/common/servlist.c:124]: File seems to use a mixture of tabs and spaces here:

image

I have changed the style to conform to those above.

Comment (dijit) [reply src/common/servlist.c:124]: wait, nevermind, I must have broken the file myself somehow. I will fix.

Comment (TingPing) [reply src/common/servlist.c:124]: You reformatted the entire file...

Comment (dijit) [reply src/common/servlist.c:124]: Yeah, I saw. Fixed immediately in https://github.com/hexchat/hexchat/commit/46fd9a460508f019b2be13edd616ef91432bd66a

**Pull request merged**